tf/variables.tf

```
@@ -0,0 +1,23 @@
+variable "aws_region" {
+  type        = string
+  description = "aws region"
+  default     = "eu-central-1"
+}
+
+variable "log_level" {
+  type        = string
+  description = "lambda extension log level"
+  default     = "trace"
+}
+
+variable "ec_region" {
```

```suggestion
variable "ess_region" {
```

For consistency with apm-server modules testing/cloud, testing/benchmark, and testing/rally-cloud.

---

Good point!